xla/service/gpu/fusions/triton/triton_fusion_emitter_large_test.cc

```diff
@@ -119,6 +119,33 @@ ENTRY e {
   EXPECT_TRUE(RunAndCompare(kHloText, ErrorSpec{/*aabs=*/1e-3, /*arel=*/1e-3}));
 }
 
+TEST_F(TritonGemmTest, MultiplePathsToSameOperandWorks) {
```

Comment:
Let's please add this test in `triton_fusion_emitter_device_legacy_test.cc` instead.

Reply:
Ok thanks for all the info! I'm not familiar with those tests and their intents so I was mostly copy pasting. I've put it in the triton_fusion_emitter_device_legacy_test.cc instead - Testing RunAndCompareNoHloPasses seems enough for detecting crush.